sdk/storage/azure-storage-blob/azure/storage/blob/blob_client.py

```diff
@@ -2095,11 +2099,11 @@ def _get_page_ranges_options( # type: ignore
             if_unmodified_since=kwargs.pop('if_unmodified_since', None),
             if_match=kwargs.pop('if_match', None),
             if_none_match=kwargs.pop('if_none_match', None))
-        page_range = None # type: ignore
-        if start_range is not None and end_range is None:
-            page_range = str(start_range)
-        elif start_range is not None and end_range is not None:
-            page_range = str(end_range - start_range + 1)
+        if length is not None:
+            length = offset + length - 1  # Reformat to an inclusive range index
+        page_range, _ = validate_and_format_range_headers(
+            offset, length, start_range_required=False, end_range_required=False, align_to_page=True
```

Comment:
So we will update this method in another PR?

Reply:
> I think there are also offset and length in File maybe Queue, so we planned to rename them in another pr right ^_^

yes, files will be updated too. I'll change validate_and_format_range_headers along with it since it's in shared folder

Reply:
BTW don't forget to edit the logic here once you rename or edit **validate_and_format_range_headers** (Though it's probably will be caught by test if you forget haha)